api/cloudcontroller/ccv3/included_resources.go

```diff
@@ -12,3 +12,14 @@ type IncludedResources struct {
 	ServicePlans     []resources.ServicePlan     `json:"service_plans,omitempty"`
 	Apps             []resources.Application     `json:"apps,omitempty"`
 }
+
```

Comment:
**suggestion** Perhaps including a small test result showing the difference in time for `cf tasks` between the old and new behaviour would be useful? Maybe with an app with like 10,000 tasks?